ansible-st2/roles/arteria_node/tasks/arteria_bcl2fastq.yml

```diff
@@ -53,5 +53,13 @@
 
 - supervisorctl:
     name: bcl2fastq-ws
-    state: started
+    state: restarted
```

Comment:
I think for later that it might be a better idea to go with `started` here, and then we should make a `notify` instead to a handler which restarts it in the event of a configuration change - this is important if we will be pushing ansible playbooks onto a running system.